backend/src/nodes/nodes/image_iterator/image_file_iterator.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import annotations
+
+import os
+from typing import Tuple, List
+
+import numpy as np
+from process import IteratorContext
+from sanic.log import logger
+
+from ...categories import ImageCategory
+from ..image.load_image import ImReadNode
+from ...node_base import IteratorNodeBase, NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import IteratorInput, DirectoryInput
+from ...properties.outputs import (
+    ImageOutput,
+    DirectoryOutput,
+    TextOutput,
+    NumberOutput,
+)
+from ...utils.image_utils import get_available_image_formats
+from ...utils.utils import numerical_sort
+
+IMAGE_ITERATOR_NODE_ID = "chainner:image:file_iterator_load"
+
+
+@NodeFactory.register(IMAGE_ITERATOR_NODE_ID)
+class ImageFileIteratorLoadImageNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = ""
+        self.inputs = [IteratorInput().make_optional()]
+        self.outputs = [
+            ImageOutput(broadcast_type=True),
+            DirectoryOutput("Image Directory"),
+            TextOutput("Subdirectory Path"),
+            TextOutput("Image Name"),
+            NumberOutput("Overall Index"),
+        ]
+
+        self.category = ImageCategory
```

Comment:
Iterators are still in the "Image" category, so could we also move them into that folder?

Reply:
Oh yeah, sure